model/src/main/scala/uk/gov/ons/addressIndex/model/server/response/AddressResponse.scala

```diff
@@ -380,161 +244,59 @@ case class AddressResponseNag(
   classificationCode: String
 )
 
-object AddressResponseNag {
-  implicit lazy val addressResponseNagFormat: Format[AddressResponseNag] = Json.format[AddressResponseNag]
+object NAG {
+  implicit lazy val fmt: Format[NAG] = Json.format[NAG]
 }
 
-/**
-  *
-  * @param paoText building name
-  * @param paoStartNumber building number
-  * @param paoStartSuffix
-  * @param paoEndNumber
-  * @param paoEndSuffix
-  */
-case class AddressResponsePao(
-  paoText: String,
-  paoStartNumber: String,
+case class PAO(
+  text: String,
+  startNumber: String,
   paoStartSuffix: String,
   paoEndNumber: String,
   paoEndSuffix: String
 )
 
-object AddressResponsePao {
-  implicit lazy val addressResponsePaoFormat: Format[AddressResponsePao] = Json.format[AddressResponsePao]
+object PAO {
+  implicit lazy val fmt: Format[PAO] = Json.format[PAO]
 }
 
-
-/**
-  *
-  * @param saoText sub building name
-  * @param saoStartNumber sub building number
-  * @param saoStartSuffix
-  * @param saoEndNumber
-  * @param saoEndSuffix
-  */
-case class AddressResponseSao(
-  saoText: String,
-  saoStartNumber: String,
-  saoStartSuffix: String,
-  saoEndNumber: String,
-  saoEndSuffix: String
+case class SAO(
+  text: String,
+  startNumber: String,
+  startSuffix: String,
+  endNumber: String,
+  endSuffix: String
 )
 
-object AddressResponseSao {
-  implicit lazy val addressResponseSaoFormat: Format[AddressResponseSao] = Json.format[AddressResponseSao]
+object SAO {
+  implicit lazy val fmt: Format[SAO] = Json.format[SAO]
 }
 
-/**
-  * Contains address geo position
-  *
-  * @param latitude  latitude
-  * @param longitude longitude
-  * @param easting   easting
-  * @param northing  northing
-  */
-case class AddressResponseGeo(
+case class GEO(
   latitude: Double,
   longitude: Double,
-  easting: Int,
-  northing: Int
+  easting: Double,
```

Comment:
They're `Doubles` in elastic, from what I can see (well `Double`s being stored as `String`s)